rosetta-source/src/main/resources/cdm-sample-files/fpml-5-13/products/dividend-swaps/expectations.json

```diff
@@ -1,7 +1,7 @@
 [
-{"fileName":"cdm-sample-files/fpml-5-13/products/dividend-swaps/div-ex01-dividend-swap.xml","excludedPaths":-1,"externalPaths":78,"outstandingMappings":26,"validationFailures":12,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"EquitySwap_ParameterReturnDividend_Index","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
-{"fileName":"cdm-sample-files/fpml-5-13/products/dividend-swaps/div-ex02-dividend-swap-collateral.xml","excludedPaths":-1,"externalPaths":97,"outstandingMappings":28,"validationFailures":14,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"EquitySwap_ParameterReturnDividend_Index","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
-{"fileName":"cdm-sample-files/fpml-5-13/products/dividend-swaps/div-ex03-dividend-swap-short-form-japanese-underlyer.xml","excludedPaths":-1,"externalPaths":87,"outstandingMappings":26,"validationFailures":8,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"EquitySwap_ParameterReturnDividend_Index","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
-{"fileName":"cdm-sample-files/fpml-5-13/products/dividend-swaps/div-ex04-dividend-swap-option-transaction-supplement.xml","excludedPaths":-1,"externalPaths":95,"outstandingMappings":27,"validationFailures":15,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"EquityOption_ParameterReturnDividend_Index","qualifiedObjectClass":"cdm.product.template.EconomicTerms"},{"qualifiedName":"EquitySwap_ParameterReturnDividend_Index","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":2}},
```

Comment:
The option no longer qualifies as a EquityOption_ParameterReturnDividend_Index which seems correct.  It seems to be a equity swaption (?) which does not have a qualification function